### PR TITLE
Update bug bounty policy with information about how to sign up for accounts

### PR DESCRIPTION
**Pull request by alsmola**
This change updates the language for the bug bounty policy and includes details about the scope of the bounty, including how to sign up for accounts.

### CODE REVIEW

File: security/bug-bounty.md [hunk 1]
```diff
@@ -1,32 +1,61 @@
...
+* https://apps.clever.com (dashboard for applications)
+* https://clever.com/oauth (instant login)
+* https://clever.com/in/ (portal)
+* https://requests.clever.com (districts request access to an app)
 
-Thank you for helping keep Clever safe!
+**Signing up for a school district account**
 
-**Thanks**
+1. Go to the [schools signup](https://schools.clever.com/signup) page.
```
Comment (dccarroll) [inline security/bug-bounty.md]: @alsmola Can we ask them to sign up at https://clever.com/signup/demo ? That will tag them as a sandbox district automatically.

File: security/bug-bounty.md [hunk 2]
```diff
@@ -1,32 +1,61 @@
...
+5. For usernames and password, select "Clever Passwords based on SIS Data (Recommended)"
+6. For student information system, select "Other" and use any name.
+7. Upload sample data through the [Web Upload](https://schools.clever.com/settings/system/upload) interface, which you can find under Settings => Sync. You can find a set of sample .csv files there.
 
-We believe in recognizing the work of others.
+**Signing up for an application account**
 
-If your work helps us improve the security of our service. we'd be happy to [acknowledge your contribution](/Clever/thanks) and ensure you are fairly rewarded for your discovery.
+1. Go to the [application developer signup](https://apps.clever.com/signup) page.
+2. Complete signup, and use an application name that starts with "#DEMO".
```
Comment (peggyl) [inline security/bug-bounty.md]: As a soft enforcement mechanism, can we note that failure to prepend the app name with `#DEMO` could result in forfeit of the bounty?

**Comment by mohit** [comment]
We should tell DS and CS about this after it's merged

File: security/bug-bounty.md [hunk 3]
```diff
@@ -1,32 +1,61 @@
...
-**Coordinated disclosure rules**
+Clever is committed to working with security experts around the world to stay up to date with the latest security techniques. If you have discovered a security issue that you believe we should know about, we'd love to hear from you. You can read about our security and privacy efforts on our [security page](https://clever.com/security).
 
+Coordinated disclosure rules
+----------------------------
 * Please let us know as soon as possible upon discovery of a potential security issue, and we’ll make every effort to quickly correct the issue.
-* Provide us a reasonable amount of time to fix the issue before publishing it elsewhere.
-* Making a good faith effort to not leak, manipulate, or destroy any user data. Please only test against accounts you own yourself or with explicit permission of the account holder.
+* Provide us with a reasonable amount of time to fix the issue before publishing it elsewhere.
+* Make a good faith effort to not leak, manipulate, or destroy any user data, or affect the availability of the site. Please only test against accounts you own yourself or with explicit permission of the account holder.
```
Comment (mohit) [inline security/bug-bounty.md]: Can we add this to eligibility also? No user data access, account takeover should be done

**Comment by mohit** [comment]
LGTM after the comments are addressed. Especially dc's

File: security/bug-bounty.md (hunk 1 above)
Comment (alsmola) [inline security/bug-bounty.md]: I wasn't familiar with this endpoint. If they created the district this way, would they still be able to create a demo app and set up instant login?

**Comment by alsmola** [comment]
@mohit I'll let District Success and Customer Success know when I've opened up the bug bounty more broadly, so that they can expect more signups from researchers following these instructions. For now, I'm just going to merge and update the private bug bounty page, since only a few people will see it.

**Comment by peggyl** [comment]
@alsmola The `dev-handbook` repo is public, so the bug bounty policy is in a sense visible to everyone.

**Comment by alsmola** [comment]
@peggyl I know, that's intentional. We're planning on publishing this on HackerOne. Did you have any concern about publishing this info?

File: security/bug-bounty.md (hunk 1 above)
Comment (dccarroll) [inline security/bug-bounty.md]: Yup!

**Pull request merged**